examples/reference-entity-model/src/main/ml-config/security/query-rolesets/pii-reader.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
I think we can get rid of this file. hubDeployAsDeveloper will generate this via hubGeneratePii anyways.

Reply:
It either needs to be gitignored or added to version control. What I'll do is make one of the properties be PII, as we really ought to have one for testing anyway. I'll give pii-reader to one of the users too.

Reply:
Just added "ssn" to Customer and made it PII. So now there's a path file and the roleset file.